### PR TITLE
feat(docs): add inline documentation to the universe package

### PR DESCRIPTION
**Pull request by sanderson**
All documentation updates in this PR have been reviewed in smaller PRs. This is the consolidated PR going into master that adds documentation to all functions in the `universe` package.

### Done checklist
- [x] docs/SPEC.md updated
- [x] Test cases written

### CODE REVIEW

File: stdlib/universe/universe.flux [hunk 1]
```diff
@@ -8,21 +14,229 @@ import "strings"
...
+//
+// ## Examples
+//
+// ### Use the current UTC time as a query boundary
+// ```no_run
+// data
+//     |> range(start: -10h, stop: now)
+// ```
+//
+// ### Define a custom now time
```
Comment (pierwill) [inline stdlib/universe/universe.flux:33]: Might want to say above (where we say the value is cached) that `now` can be overwritten.

File: stdlib/universe/universe.flux [hunk 2]
```diff
@@ -8,21 +14,229 @@ import "strings"
...
-// Transformation functions
+// chandeMomentumOscillator applies the technical momentum indicator developed
+// by Tushar Chande to input data.
+//
+// The Chande Momentum Oscillator (CMO) indicator calculates the difference
+// between the sum of all recent data points with values greater than the median
+// value of the data set and the sum of all recent data points with values lower
+// than the median value of the data set, then divides the result by the sum of
+// all data movement over a given time period.
+// It then multiplies the result by 100 and returns a value between -100 and +100.
```
Comment (pierwill) [inline stdlib/universe/universe.flux:51]: Let's split this description into multiple sentences. One approach:

```suggestion
// The Chande Momentum Oscillator (CMO) indicator 1) calculates the difference
// between the sum of all recent data points with values greater than the median
// value of the data set, 2) calculates the sum of all recent data points with values lower
// than the median value of the data set, and then 3) divides the result by the sum of
// all data movement over a given time period.
// It then multiplies the result by 100 and returns a value between -100 and +100.
```

What is "the result" in number 3 above?

File: stdlib/universe/universe.flux [hunk 3]
```diff
@@ -8,21 +14,229 @@ import "strings"
...
+//
+// #### Output tables
+// For each input table with x rows, `chandeMomentumOscillator()` outputs a
```
Comment (pierwill) [inline stdlib/universe/universe.flux:54]:
```suggestion
// For each input table with `x` rows, `chandeMomentumOscillator()` outputs a
```

File: stdlib/universe/universe.flux [hunk 4]
```diff
@@ -8,21 +14,229 @@ import "strings"
...
 import "experimental/table"
 
-// now is a function option whose default behaviour is to return the current system time
+// now is a function option that, by default, returns the current system time.
+//
+// #### now() vs system.time()
+// `now()` returns the current system time (UTC). `now()` is cached at runtime,
+// so all executions of `now()` in a Flux script return the same time value.
+// `system.time()` returns the system time (UTC) at which `system.time()` is executed.
+// Each instance of `system.time()` in a Flux script returns a unique value.
```
[the remark this answers is not in the document]
Comment (sanderson) [reply stdlib/universe/universe.flux:23]: Maybe... "guaranteed" is a strong word.

File: libflux/flux-core/src/bin/fluxdoc.rs [hunk 5]
```diff
@@ -389,6 +389,5 @@ const EXCEPTIONS: &[&str] = &[
     "testing/prometheus",
     "testing/promql",
     "testing/usage",
-    "universe",
```
Comment (nathanielc) [inline libflux/flux-core/src/bin/fluxdoc.rs:392]: Love it!

**Review by nathanielc — APPROVED** [review]
Looks great, I followed the intermediate PRs as they merged in, approving this now

**Pull request merged**